### DISCUSSION TITLE
Increaseing the speed of execution

### DISCUSSION THREAD

**Discussion by Lahari-77** [PaddlePaddle/PaddleOCR#10712] [Q&A]
Hi,
I am using nvidia-smi driver version 515
Cuda tool kit :11.7
Paddlepaddlegpu:2.4.2
I have a program which has 6 loops.Each time inside the loop paddleocr has to extract text.
Even after enabling gpu=True.Each loop i staking 2 seconds.Resulting in 12 seconds.Which is very bad according to the performance parameters.What can be done do cut down the 12 sec to atleast 2 3 seconds.

**Comment by tzktok** [comment]
find any solution @Lahari-77 ?

**Reply by Lahari-77** [reply]
Its because every time i used to run my program i used to load the model.What you have to do is just load model once and run the program.

**Reply by tzktok** [reply]
I have initialize the model first itself..bt every time model loads taks time.. @Lahari-77

**Reply by GreatV** [reply]
Try using the Paddle dev version.

**Reply by tzktok** [reply]
> > Try using the Paddle dev version.

 what's the different between stable nd dev version of paddle? @GreatV

**Reply by GreatV** [reply]
@tzktok https://www.paddlepaddle.org.cn/en/install/quick?docurl=/documentation/docs/en/develop/install/pip/linux-pip_en.html

**Reply by GreatV** [reply]
The new version of paddle has many more new features and fixes.

**Comment by Lahari-77** [comment]
May be u r loading and  running at the same instance.Try loading the model while starting tjhe server and then try paddle.

**Reply by tzktok** [reply]
> May be u r loading and running at the same instance.Try loading the model while starting tjhe server and then try paddle.

paddleocr start time in gpu is almost 6sec..how to reduce this time.. @Lahari-77  @GreatV
